### PR TITLE
Release notes for Edge 2.2.3.0

### PR DESCRIPTION
**Pull request by usha-mandya**
### Proposed changes

Release notes for Docker Desktop 2.2.3.0

### Unreleased project version (optional)
2.2.3.0

### Related issues (optional)

https://docker.atlassian.net/browse/DESKTOP-2581

### CODE REVIEW

File: docker-for-windows/edge-release-notes.md [hunk 1]
```diff
@@ -12,6 +12,57 @@ For information about Stable releases, see the [Stable release
...
+- [Docker 19.03.8](https://github.com/docker/docker-ce/releases/tag/v19.03.8)
+- [Docker Compose 1.26.0-rc3](https://github.com/docker/compose/releases/tag/1.26.0-rc3)
+- [Linux 4.19.76](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-4e5d9e5f3bde0abf236f97e4a81b029ae0f5f6e7-amd64/images/sha256-11dc0f6ee3187088219ba1463ebb378f5093a7d98f176ddfd62dd6b741c2dd2d?context=repo)
+
+### New
+
+- Docker Desktop introduces a new onboarding tutorial upon first startup. The Quick Start tutorial guides users to get started with Docker in a few easy steps. It includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
+
+### Bug fixes and minor changes
+
```
Comment (nebuk89) [inline docker-for-windows/edge-release-notes.md]: Should we maybe include a total 'count' of the number of public bugs we have fixed. Just to improve that 'why should I upgrade' moment for a user?

File: docker-for-mac/edge-release-notes.md [hunk 2]
```diff
@@ -12,6 +12,32 @@ For information about Stable releases, see the [Stable release
...
+- [Docker Compose 1.26.0-rc3](https://github.com/docker/compose/releases/tag/1.26.0-rc3)
+- [Linux 4.19.76](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-4e5d9e5f3bde0abf236f97e4a81b029ae0f5f6e7-amd64/images/sha256-11dc0f6ee3187088219ba1463ebb378f5093a7d98f176ddfd62dd6b741c2dd2d?context=repo)
+
+### New
+
+- Docker Desktop introduces a new onboarding tutorial upon first startup. The Quick Start tutorial guides users to get started with Docker in a few easy steps. It includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
+
+### Bug fixes and minor changes
+
+- Added an option to delete data from the **Troubleshoot** screen.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Good point. Should this be `Container data` @mat007?

File: docker-for-windows/edge-release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: Sure. Perhaps on the lines of 

`Docker Desktop Edge 2.2.3.0 fixes nn bugs reported on the docker/for-win and docker/for-mac GitHub repositories.`

@mat007 Would it be possible to get the number of bugs we've fixed on Edge, please? TIA

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mat007) [reply docker-for-mac/edge-release-notes.md]: It’s everything so `container and image data`.

File: docker-for-windows/edge-release-notes.md (hunk 1 above)
Comment (mat007) [reply docker-for-windows/edge-release-notes.md]: Would summing up the number of links to `for-win` and `for-mac` tickets work for you @nebuk89 ?

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (mat007) [inline docker-for-mac/edge-release-notes.md]: That’s only on Windows at the moment, so should not be listed in the Mac notes.

File: docker-for-windows/edge-release-notes.md [hunk 3]
```diff
@@ -12,6 +12,57 @@ For information about Stable releases, see the [Stable release
...
+- Fixed a bug where files did not change in shared volumes when using `volumes_from` in docker-compose.yml. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
+
+**Other fixes**
+- Added an option to delete data from the **Troubleshoot** screen.
+- Docker Desktop now reads the Hyper-V VM disk max size on startup and uses it as the value to display in the Settings.
+- Fixed the Hyper-V used disk image size being reported incorrectly.
+- Fixed the Hyper-V VM disk size increase. Fixes [docker/for-win#5881](https://github.com/docker/for-win/issues/5881).
+- Fixed container time drift when system hibernates. Fixes [docker/for-win#4526](https://github.com/docker/for-win/issues/4526).
+- Fixed a bug where the Docker Desktop UI could be started without the engine. Fixes [docker/for-win#5376](https://github.com/docker/for-win/issues/5376).
+- Docker Desktop now uses the least possible privilege for querying the LanManServer service. Fixes [docker/for-win#5150](https://github.com/docker/for-win/issues/5150).
```
Comment (mat007) [inline docker-for-windows/edge-release-notes.md]: It’s `LanmanServer`

image

but we could probably should say `Server service`.

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: Sure, thanks. Will remove from Mac release notes.

File: docker-for-windows/edge-release-notes.md (hunk 3 above)
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: Thanks. Will update it as `Server service`.

File: docker-for-windows/edge-release-notes.md (hunk 1 above)
Comment (nebuk89) [reply docker-for-windows/edge-release-notes.md:31]: Yes exactly that is what customers would see as the 'number of fixes' and I think would be great!

Comment (mat007) [reply docker-for-windows/edge-release-notes.md:31]: Or we could even just count the number of entries in «Bug fixes and minor changes», that’s 7 for the Mac and 28 for Windows!

File: docker-for-mac/edge-release-notes.md [hunk 4]
```diff
@@ -12,6 +12,33 @@ For information about Stable releases, see the [Stable release
...
+- [Docker Compose 1.26.0-rc3](https://github.com/docker/compose/releases/tag/1.26.0-rc3)
+- [Linux 4.19.76](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-4e5d9e5f3bde0abf236f97e4a81b029ae0f5f6e7-amd64/images/sha256-11dc0f6ee3187088219ba1463ebb378f5093a7d98f176ddfd62dd6b741c2dd2d?context=repo)
+
+### New
+
+- Docker Desktop introduces a new onboarding tutorial upon first startup. The Quick Start tutorial guides users to get started with Docker in a few easy steps. It includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
+
+### Bug fixes and minor changes
+
+> Docker Desktop Edge 2.2.3.0 fixes 7 issues reported on the [docker/for-mac](https://github.com/docker/for-mac/issues) GitHub repository.
```
Comment (usha-mandya) [inline docker-for-mac/edge-release-notes.md:32]: @nebuk89 @mat007 WDYT?

image

File: docker-for-windows/edge-release-notes.md [hunk 5]
```diff
@@ -12,6 +12,59 @@ For information about Stable releases, see the [Stable release
...
+- [Docker Compose 1.26.0-rc3](https://github.com/docker/compose/releases/tag/1.26.0-rc3)
+- [Linux 4.19.76](https://hub.docker.com/layers/docker/for-desktop-kernel/4.19.76-4e5d9e5f3bde0abf236f97e4a81b029ae0f5f6e7-amd64/images/sha256-11dc0f6ee3187088219ba1463ebb378f5093a7d98f176ddfd62dd6b741c2dd2d?context=repo)
+
+### New
+
+- Docker Desktop introduces a new onboarding tutorial upon first startup. The Quick Start tutorial guides users to get started with Docker in a few easy steps. It includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
+
+### Bug fixes and minor changes
+
+> Docker Desktop Edge 2.2.3.0 fixes 28 issues reported on the [docker/for-win](https://github.com/docker/for-win/issues) GitHub repository.
```
Comment (usha-mandya) [inline docker-for-windows/edge-release-notes.md:32]: @nebuk89 @mat007 WDYT?

image

**Pull request merged**